generate_brick.py

```diff
@@ -268,21 +331,38 @@ def define_properties(definitions, superprop=None):
 G.add((BRICK.Quantity, RDFS.subClassOf, SOSA.ObservableProperty))
 G.add((BRICK.Quantity, RDFS.subClassOf, BRICK.Measurable))
 G.add((BRICK.Quantity, A, OWL.Class))
+G.add((BRICK.Quantity, RDFS.subClassOf, SKOS.Concept))
 # set up Substance definition
 G.add((BRICK.Substance, RDFS.subClassOf, SOSA.FeatureOfInterest))
 G.add((BRICK.Substance, RDFS.subClassOf, BRICK.Measurable))
 G.add((BRICK.Substance, A, OWL.Class))
+G.add((BRICK.Substance, RDFS.subClassOf, SKOS.Concept))
```

Comment:
Is Substance a skos:Concept too? I can't find a description of this change.

Reply:
I think I added this by mistake --- only looking at Brick Quantity for now. I'll remove it